specification/marketplacecatalog/resource-manager/Microsoft.Marketplace/preview/2022-09-01-preview/marketplacecatalog.json

```diff
@@ -0,0 +1,1240 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Catalog Browse API",
+    "description": "Microsoft Marketplace Catalog Browse API",
+    "version": "2022-09-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Marketplace/products/{id}": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=6&length=13)

There are a bunch of breaking changes being flagged. Please take a look and address. Once addressed, please remove the ARMChangesRequested label from the PR .

Reply:
Hi rkmanda,
The results are different as this api-version introduces a new returned object schema. I am a little unclear as to why this is considered a breaking change as the schema change is done under an entirely new api-version.